[2022-07-28 15:09:27,029] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: GME_historic_data.create_historic_table scheduled__2022-07-28T14:30:00+00:00 [queued]>
[2022-07-28 15:09:27,052] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: GME_historic_data.create_historic_table scheduled__2022-07-28T14:30:00+00:00 [queued]>
[2022-07-28 15:09:27,054] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 15:09:27,054] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-28 15:09:27,055] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 15:09:27,085] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): create_historic_table> on 2022-07-28 14:30:00+00:00
[2022-07-28 15:09:27,091] {standard_task_runner.py:52} INFO - Started process 748 to run task
[2022-07-28 15:09:27,098] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'GME_historic_data', 'create_historic_table', 'scheduled__2022-07-28T14:30:00+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/open_options.py', '--cfg-path', '/tmp/tmp1pa1bbob', '--error-file', '/tmp/tmpfe2d6p1p']
[2022-07-28 15:09:27,100] {standard_task_runner.py:80} INFO - Job 83: Subtask create_historic_table
[2022-07-28 15:09:27,235] {task_command.py:371} INFO - Running <TaskInstance: GME_historic_data.create_historic_table scheduled__2022-07-28T14:30:00+00:00 [running]> on host a2299eb83309
[2022-07-28 15:09:27,452] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=vardhan
AIRFLOW_CTX_DAG_ID=GME_historic_data
AIRFLOW_CTX_TASK_ID=create_historic_table
AIRFLOW_CTX_EXECUTION_DATE=2022-07-28T14:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-28T14:30:00+00:00
[2022-07-28 15:09:27,475] {base.py:68} INFO - Using connection ID 'postgres_optionsdata' for task execution.
[2022-07-28 15:09:27,479] {dbapi.py:231} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS historic__GME_ (
            datetime BIGINT,
            high DOUBLE PRECISION,
            low DOUBLE PRECISION,
            open DOUBLE PRECISION,
            close DOUBLE PRECISION,
            volume INTEGER,
            symbol VARCHAR(32) NOT NULL,
            PRIMARY KEY (symbol, datetime)
        )
    , parameters: None
[2022-07-28 15:09:27,495] {python.py:173} INFO - Done. Returned value was: None
[2022-07-28 15:09:27,531] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=GME_historic_data, task_id=create_historic_table, execution_date=20220728T143000, start_date=20220728T150927, end_date=20220728T150927
[2022-07-28 15:09:27,593] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-28 15:09:28,007] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
